app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutTreatments.java

```diff
@@ -200,8 +204,59 @@ public static boolean processTreatmentResponse(final String response) throws Exc
                         }
                     }
                 }
+            } else {
+                if(tr.getString("eventType").indexOf("Temp Basal") != -1) {
+                    new_data |= HandleTempBassal(tr);
+                }
             }
         }
         return new_data;
     }
+    private static boolean HandleTempBassal(JSONObject tr) throws JSONException {
+        // Code here is trying to do what the nightscout code does for this kind of bassals.
```

Comment:
bassals -> basals

Please turn this comment into a Java Docs function description.